apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema banking.BankAccount 'organization:Relationship:Organization:CASCADE; tin:Text; country:Text; routingNumber:Text; number:Text; currency:Text; approvedAt?:DateTimeUtc; approvedBy?:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
+ */
+
+const { get } = require('lodash')
+const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
+
+/**
+ * BankAccount entity can be read either by:
+ * 1. By admin or support
+ * 2. Organization employee (without settings / state)
+ */
+async function canReadBankAccounts ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    return { organization: { employees_some: { user: { id: user.id },  isBlocked: false } } }
+}
+
+async function canManageBankAccounts ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    return false
+}
+
+async function canManageIsApprovedField ({ authentication: { item: user }, originalInput }) {
+    if (user.isAdmin || user.isSupport) return true
+
+    // If user is not support and admin, then he only can drop isApproved fields
+    if (!get(originalInput, 'approvedAt') && !get(originalInput, 'approvedBy')) {
```

Comment:
If you want to inspect the presence of only one key, let's do it explicitly:

```suggestion
    // Allow only to change `isApproved` field
    if ('isApproved' in originalInput && Object.keys(originalInput).length ==== 1) {
```